.github/.bc-linter.yml

```diff
@@ -0,0 +1,24 @@
+# doc: https://github.com/pytorch/test-infra/blob/main/tools/stronghold/docs/bc_linter_config.md
```

Comment:
flagging that currently bc-linter looks for config only in the root directory, might need to add support for alternative directories before merging this PR.